sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql

```diff
@@ -60,15 +62,18 @@ SELECT
   stats.impressions AS ad_impressions,
   install_dou_metrics.new_profiles_sum AS installs,
   install_dou_metrics.dau_sum AS dous,
+  install_dou_metrics.dau_sum AS dau,
   stats.conversions AS ad_conversions,
   stats.marketing_ad_clicks AS marketing_ad_clicks,
   install_dou_metrics.revenue_generating_ad_clicks_sum AS revenue_generating_ad_clicks,
+  install_dou_metrics.new_installs_sum,
+  install_dou_metrics.activations_sum,
   -- Cost per-install for each campaign ($/new profiles)
   CASE
     WHEN install_dou_metrics.new_profiles_sum = 0
       THEN 0
     ELSE stats.cost_micros / install_dou_metrics.new_profiles_sum
-  END AS cost_per_install_micros,
+  END AS cost_per_new_profile_micros,
```

Comment:
Ah, renamed here so we can add the metric on line 98 by the original name, gotcha